dvc/exceptions.py

```diff
@@ -2,9 +2,11 @@
 
 from __future__ import unicode_literals
 
+import traceback
+
 from dvc.utils.compat import str, builtin_str
 
-import traceback
+
```

Comment:
Please delete this line. No need for three lines.

Reply:
done.